### DISCUSSION TITLE
Can svg file be previewed in yazi?

### DISCUSSION THREAD

**Discussion by OriLiMu** [sxyazi/yazi#1308] [1. Q&A]
I check the doc seems not found related

**Comment by xfzv** [comment]
Yes, works fine here. You need to have `imagemagick` installed. Ensure you're using the following in `previewers` section of `yazi.toml`:

https://github.com/sxyazi/yazi/blob/f0cb365839c94690612a34ed9fd7edd71fde9fa1/yazi-config/preset/yazi.toml#L108
